### PR TITLE
[DOCS] Update how to configure an expectation store in GCS

### PR DESCRIPTION
**Pull request by NathanFarmer**
Changes proposed in this pull request:
- GREAT-404
- Subtask: GREAT-408
- Bring documentation for "How to configure an expectation store in GCS" under test
- Add Google Cloud SDK to `docs-integration` pipeline for use with `gcloud` and `gsutil` CLI commands

### Definition of Done
- [X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [X] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [X] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ ipywidgets>=7.5.1  # package
 jinja2>=2.10  # package
 jsonpatch>=1.22  # package
 jsonschema>=2.5.1  # package
-mistune>=0.8.4  # package
+mistune>=0.8.4,<2.0.0  # package
```
Comment (NathanFarmer) [inline requirements.txt:14]: The latest `mistune` (2.0.0) is incompatible with the latest `nbconvert` (6.3.0)

File: azure-pipelines-docs-integration.yml [hunk 2]
```diff
@@ -94,6 +95,16 @@ stages:
...
               retryCount: '2'
+
+        - script: |
+            curl -sS https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-367.0.0-linux-x86_64.tar.gz > ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz && tar -xf ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz
+            ./google-cloud-sdk/install.sh --usage-reporting=False --path-update=True --quiet --install-python=False
+            ./google-cloud-sdk/bin/gcloud config configurations create superconductive-internal-configurations
+            ./google-cloud-sdk/bin/gcloud config set account sandbox-account-for-azure-tests --project=superconductive-internal --access-token-file=$(gcp_authkey.secureFilePath)
+            ./google-cloud-sdk/bin/gcloud config set pass_credentials_to_gsutil True
+            ./google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=$(gcp_authkey.secureFilePath)
+          displayName: 'Install Google Cloud SDK'
```
Comment (NathanFarmer) [inline azure-pipelines-docs-integration.yml:106]: Google Cloud SDK is required to test `gsutil` and could be leveraged in future docs that demonstrate `gcloud` commands.

File: requirements.txt (hunk 1 above)
Comment (NathanFarmer) [inline requirements.txt:14]: I was unable to figure out how to un-pin `pip` and resolve `aiobotocore` dependencies. The pinned version of `pip` needs to know about this constraint.

File: azure-pipelines-docs-integration.yml (hunk 2 above)
Comment (Shinnnyshinshin) [reply azure-pipelines-docs-integration.yml:106]: amazing. Thank you for including this 🙇🏼

File: tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+with open(great_expectations_yaml_file_path, "w") as f:
+    yaml.dump(great_expectations_yaml, f, default_flow_style=False)
+
+expectation_suite_name = "my_expectation_suite"
+context.create_expectation_suite(expectation_suite_name=expectation_suite_name)
+
+# try gsutil cp command
+copy_expectation_command = """
+gsutil cp expectations/my_expectation_suite.json gs://<YOUR GCS BUCKET NAME>/<YOUR GCS PREFIX NAME>/my_expectation_suite.json
+"""
```
Comment (Shinnnyshinshin) [inline tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py:88]: wow we are actually doing this? Love it

File: tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    check=True,
+    stderr=subprocess.PIPE,
+)
+stderr = result.stderr.decode("utf-8")
+
+copy_expectation_output = """
+Operation completed over 1 objects
+"""
+
+assert copy_expectation_output.strip() in stderr
```
Comment (Shinnnyshinshin) [inline tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py:121]: Wow

**Review by Shinnnyshinshin — APPROVED** [review]
I'm blown away by this @NathanFarmer. The script is really expanding what is possible with our docs integration tests .. Thank you for the thought and work that has obviously gone into this PR

File: azure-pipelines-docs-integration.yml (hunk 2 above)
Comment (alexsherstinsky) [reply azure-pipelines-docs-integration.yml:106]: ❤️

File: tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+)
+with open(great_expectations_yaml_file_path, "r") as f:
+    great_expectations_yaml = yaml.safe_load(f)
+
+stores = great_expectations_yaml["stores"]
+pop_stores = ["checkpoint_store", "evaluation_parameter_store", "validations_store"]
+for store in pop_stores:
+    stores.pop(store)
+
+actual_existing_expectations_store = dict()
```
Comment (alexsherstinsky) [inline tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py]: @NathanFarmer we should use `{}` universally, unless concatenating several dictionaries into one.  Thanks.

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Small change request; otherwise, awesome!

File: tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py (hunk 5 above)
Comment (NathanFarmer) [reply tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py]: ✅

File: tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py (hunk 3 above)
Comment (NathanFarmer) [reply tests/integration/docusaurus/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.py:88]: Yes sir we are!

**Review by alexsherstinsky — APPROVED** [review]
LGTM!

**Pull request merged**